packages/rancher-vsphere/rancher-vsphere-csi/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: local
-version: 100.1.0+up2.3.0
```

Comment:
It might be better to do `100.1.1+up2.3.0` since it's a small change. We normally bump the major number when a  new Rancher version is released, the minor number when we update the chart based on the upstream or introduce significant/breaking changes, and the patch for everything else.

Reply:
Fixed!